core/unit_test/CMakeLists.txt

```diff
@@ -206,6 +212,12 @@ if(Kokkos_ENABLE_OPENMP)
       UnitTestMain.cpp
       openmp/TestOpenMP_InterOp.cpp
   )
+  KOKKOS_ADD_EXECUTABLE_AND_TEST(
+    UnitTest_OpenMPGraph
+    SOURCES
+      UnitTestMainInit.cpp
+      openmp/TestOpenMP_Graph.cpp
+  )
 endif()
```

Comment:
since the default should work for everything I think we could just add this unconditionally to the normal tests?

Reply:
I _really_ would prefer not to add to the absolute disaster that is "put every single test in a single executable and take way to long to compile for a tiny change." Is there a reason to keep adding to this disaster rather than starting to split off features into their own tests?